code/modules/spells/spell_types/pointed/duffelbag_curse.dm

```diff
@@ -0,0 +1,52 @@
+/obj/effect/proc_holder/spell/pointed/duffelbagcurse
+	name = "Duffelbag Curse"
+	desc = "A spell that summons a duffelbag demon in the target's back, slowing him down and slowly eating him."
+	school = "transmutation"
+	charge_type = "recharge"
+	charge_max	= 80
+	charge_counter = 0
+	clothes_req = FALSE
+	stat_allowed = FALSE
+	invocation = "BA'R A'RP!"
+	invocation_type = INVOCATION_SHOUT
+	range = 7
+	cooldown_min = 20
+	ranged_mousepointer = 'icons/effects/mouse_pointers/mecha_mouse.dmi'
+	action_icon_state = "duffelbag_curse"
```

Comment:
```suggestion
	action_icon_state = "duffelbag_curse"
```
the snakecase for this and the thing it's referencing should be changed to "duffel_bag_curse"